webapp/src/main/resources/public/js/components/users/UserDeleteModal.js

```diff
@@ -0,0 +1,70 @@
+import React from "react";
+import {FormattedMessage, intlShape, injectIntl} from "react-intl";
+import {Button, Modal} from "react-bootstrap";
+import FluxyMixin from "alt-mixins/FluxyMixin";
+import UserStore from "../../stores/users/UserStore";
+import UserActions from "../../actions/users/UserActions";
+
+
+let createClass = require('create-react-class');
+
+let UserDeleteModal = createClass({
+
+    mixins: [FluxyMixin],
+
+    propTypes: {
+        intl: intlShape.isRequired,
+    },
+
+    statics: {
+        storeListeners: {
+            "onUserStoreChanged": UserStore,
```

Comment:
In recent code, there was an attempt to move to stateless react component and the state passed down from the alt store. See BranchesPage/BranchesSearchResults/etc or ScreenshotsPage/ScreenshotsGrid/etc

You may want to consider using it for those new pages too. It might be easier to swap out "alt" later which is an outdated library (considering redux).

Reply:
I can look into it.

Do you also want completely stateless components (so no `getInitialState` and no `this.setState({...})`), or just how the existing alt stores are used? Especially looking at the `UserModal`, moving the state into a dedicated store would increase the code size quite a bit (dedicated actions, a dedicated store).

Reply:
Right it requires adding a store and having callbacks passed as props in the components. I found it relatively clean in the end but I haven't touch the frontend in a while, I can't say how much work it is to convert this. 

Since it works as it, I'm fine with the current code if it is too much to migrate.

Reply:
I have refactored the frontend to now use `AltContainer`.

Reply:
Thanks a lot, it looks great! I added a minor comment related to passing the actions down via props.